packages/metro/src/lib/contextModuleTemplates.js

```diff
@@ -34,6 +34,10 @@ function createFileMap(
       if (!filePath.startsWith('.')) {
         filePath = `.${path.sep}` + filePath;
       }
+      // NOTE(byCedric): On Windows, normalize the backslashes to forward slashes to match the behavior in Webpack.
+      if (path.sep === '\\') {
+        filePath = filePath.replace(/\\/g, '/');
```

Comment:
We are using npm package slash for this usecase, better reuse the same. 

```suggestion
        filePath = slash(filePath);
```

Reply:
Note that we've used `slash` as a `devDependency` within `metro-file-map`, and it is only referenced once. Therefore probably *not* worth including it here — `filePath.replaceAll(path.sep, '/')` should do.

Reply:
Alright, let me revert that.

Reply:
Done